l10n/po/pt_BR/_guides/kafka-dev-ui.adoc.po

```diff
@@ -3,84 +3,77 @@
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.3.2\n"
 
 #. This guide is maintained in the main Quarkus repository
 #. and pull requests should be submitted there:
 #. https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 #. type: Title =
 #: upstream/_guides/kafka-dev-ui.adoc:6
-#, fuzzy, no-wrap
+#, no-wrap
 msgid "Kafka Dev UI"
-msgstr "Interface de desenvolvimento do Kafka"
+msgstr "Kafka Dev UI"
 
 #. type: Plain text
 #: upstream/_guides/kafka-dev-ui.adoc:14
-#, fuzzy
 msgid "If any Kafka-related extension is present (e.g. `quarkus-smallrye-reactive-messaging-kafka`), the Quarkus Dev UI is extended with a Kafka broker management UI.  It is connected automatically to the Kafka broker configured for the application."
-msgstr "Se estiver presente uma extensão relacionada com o Kafka (por exemplo, `quarkus-smallrye-reactive-messaging-kafka`), a interface de utilizador do Quarkus Dev é alargada com uma interface de gestão do corretor Kafka. Esta é ligada automaticamente ao corretor Kafka configurado para a aplicação."
+msgstr "Se estiver presente uma extensão relacionada com o Kafka (por exemplo, `quarkus-smallrye-reactive-messaging-kafka`), a Quarkus Dev UI é estendida com uma interface de gestão do corretor Kafka. Esta é ligada automaticamente ao corretor Kafka configurado para a aplicação."
 
 #. type: Named 'alt' AttributeList argument for macro 'image'
 #: upstream/_guides/kafka-dev-ui.adoc:15 upstream/_guides/kafka-dev-ui.adoc:24
-#, fuzzy, no-wrap
+#, no-wrap
 msgid "Kafka"
 msgstr "Kafka"
 
 #. type: Target for macro image
 #: upstream/_guides/kafka-dev-ui.adoc:15
 #, no-wrap
 msgid "kafka-dev-ui-link.png"
-msgstr ""
+msgstr "kafka-dev-ui-link.png"
 
 #. type: Plain text
 #: upstream/_guides/kafka-dev-ui.adoc:18
-#, fuzzy
 msgid "With the *Kafka Dev UI*, you can directly manage your Kafka cluster and perform tasks, such as:"
 msgstr "Com o *Kafka Dev UI*, pode gerir diretamente o seu cluster Kafka e executar tarefas, tais como:"
 
 #. type: Plain text
 #: upstream/_guides/kafka-dev-ui.adoc:20
-#, fuzzy
 msgid "Listing and creating topics"
 msgstr "Listar e criar tópicos"
 
 #. type: Plain text
 #: upstream/_guides/kafka-dev-ui.adoc:21
-#, fuzzy
 msgid "Visualizing records"
-msgstr "Visualização de registos"
+msgstr "Visualização de registros"
 
 #. type: Plain text
 #: upstream/_guides/kafka-dev-ui.adoc:22
-#, fuzzy
 msgid "Publishing new records"
-msgstr "Publicação de novos registos"
+msgstr "Publicação de novos registros"
 
 #. type: Plain text
 #: upstream/_guides/kafka-dev-ui.adoc:23
-#, fuzzy
 msgid "Inspecting the list of consumer groups and their consumption lag"
 msgstr "Inspeção da lista de grupos de consumidores e do respetivo desfasamento de consumo"
 
 #. type: Target for macro image
 #: upstream/_guides/kafka-dev-ui.adoc:24
 #, no-wrap
 msgid "kafka-dev-ui-records.png"
-msgstr ""
+msgstr "kafka-dev-ui-records.png"
```

Comment:
No need to repeat if the content doesn't change, but that's ok

Reply:
ok, this is my first contribution to open source, I apologize, but I will try my best to do my best